src/pages/docs/step-ca/configuration.mdx

```diff
@@ -510,23 +510,67 @@ At this time `step-ca` does not have any API or interface for easily exporting d
 
 ## Cryptographic Protection
 
-By default, `step-ca` stores its signing keys encrypted on disk, on the CA.
-Some use cases may require more advanced cryptographic protection.
-For protection of you signing keys,
-`step-ca` integrates with [Google Cloud KMS](#google-cloud-kms),
-[AWS KMS](#aws-kms),
-[Azure Key Vault](#azure-key-vault), 
-[YubiKey PIV](#yubikey-piv),
-and [PKCS #11 hardware security modules (HSMs)](#pkcs-11).
-
-For a complete, end-to-end tutorial using YubiKeys,
+By default, `step-ca` stores its signing keys encrypted on disk.
+Some use cases may require more advanced cryptographic protection (or hardware protection)
+of CA's signing keys.
+For these scenarios, `step-ca` integrates with the following key management systems:
+
+- [Google Cloud KMS](#google-cloud-kms)
+- [AWS KMS](#aws-kms)
+- [Azure Key Vault](#azure-key-vault)
+- [YubiKey PIV](#yubikey-piv)
+- [PKCS #11 hardware security modules (HSMs)](#pkcs-11)
+- ssh-agent
+
+For a complete, end-to-end example using a YubiKey,
 see our blog post [Build a Tiny Certificate Authority For Your Homelab](https://smallstep.com/blog/build-a-tiny-ca-with-raspberry-pi-yubikey/).
 
+### Before You Begin
+
+The `step ca init` command does not have KMS support; it only supports creating keys on disk.
+
+We've created the [`step kms` plugin](https://github.com/smallstep/step-kms-plugin) for managing the keys and certificates on cloud KMSs and on hardware devices. You'll need to install this plugin before continuing with any of the examples below.
+
 ### Google Cloud KMS
 
 [Cloud KMS](https://cloud.google.com/kms/docs) is Google's cloud-hosted KMS that allows you to store the cryptographic keys and sign certificates using their infrastructure. Cloud KMS supports two different protection levels: `SOFTWARE` and `HSM`.
 
-To configure Cloud KMS in your certificate authority, add the `kms` object to your `ca.json` file and replace the property `key` with the Cloud KMS key name of your intermediate key:
+First, let's create your X.509 CA keys inside Google Cloud KMS. Run:
+
+```shell nocopy
+$ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
+$ step kms create 'cloudkms:credentials-file=/path/to/kms-credentials.json;object=root'
```

Comment:
If you export that variable, the credentials-file attribute might not be necessary. Can you check?

Reply:
The same should be true for next lines. But you still need `--kms 'cloudkms:'`.

Reply:
I'll check this out, thanks